tableau_twitter.py

```diff
@@ -1,254 +1 @@
-# -*- coding: utf-8 -*-
 
-from tweepy import Stream
-from tweepy import OAuthHandler
-from tweepy import API
-from tweepy.streaming import StreamListener
-import csv
-import json
-from keys import keys
-import os
-import dataextract as tde
-import re
-from datetime import datetime
-from textblob import TextBlob
-import time
-
-# put your keys here. Get them by registering an app on dev.twitter.com
-SCREEN_NAME = keys['screen_name']
```

Comment:
Am I supposed to put the keys directly into the code? i recommend reading in from a .twitterrc file  like this: https://github.com/stevenqzhang/bmndr/blob/master/bmndr#L20